identity/manager.go

```diff
@@ -51,6 +54,37 @@ func (idm *identityManager) GetIdentities() []Identity {
 	return ids
 }
 
+func (idm *identityManager) Unlock(identity Identity, passphrase string) error {
+	keystoreManager := idm.keystoreManager
+	accountExisting := identityToAccount(identity)
+
+	account, err := keystoreManager.Find(accountExisting)
+	if err != nil {
+		return err
+	}
+
+	err = keystoreManager.Unlock(account, passphrase)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (idm *identityManager) IsUnlocked(identity Identity) bool {
+	_, err := idm.keystoreManager.SignHash(identityToAccount(identity), messageHash([]byte("1")))
```

Comment:
- Strange that readonly function does write action
- I would skip implementing `IsUnlocked` flag without real usecase

Reply:
I agree, it just seemed like we will need something like this and this will be more friendly than err != keystore.ErrLocked. Plus we won't need to use keystore package externally for these checks:)

Reply:
And if finally we do have `isLocked` flag, we should read it from Ethereum library see `keystoreWallet.Status()`

Reply:
If this IsUnlocked functionality already in wallet then we are good to go.
Use case:
1. Consumer gets list of identities from tequila endoint
2. Consumer needs to decide some how if he needs to unlock identity he wants to use for connection. Calling unlock everytime when he wants to use it - sound a bit overhead? lock-status subresource with locked : true| false would be a nice indication in tequila endpoint

Reply:
keystoreWallet is private to the keystore package as far as I can see. Have you seen it exposed through some function?